docker/actinia-plugin-tests/actinia-test.cfg

```
@@ -0,0 +1,43 @@
+[GRASS]
+grass_gis_start_script = /usr/local/bin/grass
+grass_database = /actinia_core/grassdb
+grass_user_database = /actinia_core/userdata
+grass_tmp_database = /actinia_core/workspace/temp_db
+grass_resource_dir = /actinia_core/resources
+grass_addon_path = /root/.grass7/addons/
+grass_gis_base = /usr/local/grass7
+grass_modules_xml_path = /usr/local/grass7/gui/wxpython/xml/module_items.xml
```

```suggestion
grass_addon_path = /root/.grass8/addons/
grass_gis_base = /usr/local/grass8
grass_modules_xml_path = /usr/local/grass8/gui/wxpython/xml/module_items.xml
```
Don't we use GRASS GIS 8?